diff -uraN gnome-builder-43.2/src/plugins/meson-templates/resources/src/main-gtk4.py omv-gnome-builder-43.2/src/plugins/meson-templates/resources/src/main-gtk4.py
--- gnome-builder-43.2/src/plugins/meson-templates/resources/src/main-gtk4.py	2022-10-05 02:10:26.000000000 +0200
+++ omv-gnome-builder-43.2/src/plugins/meson-templates/resources/src/main-gtk4.py	2022-10-10 20:46:15.113774535 +0200
@@ -4,11 +4,9 @@
 import gi
 
 gi.require_version('Gtk', '4.0')
-{{if is_adwaita}}
 gi.require_version('Adw', '1')
-{{end}}
 
-from gi.repository import Gtk, Gio{{if is_adwaita}}, Adw{{end}}
+from gi.repository import Gtk, Gio, Adw
 
 from .window import {{PreFix}}Window
 
